onnxruntime/hosting/server_configuration.h

```diff
@@ -54,18 +57,22 @@ class ServerConfiguration {
   const std::string full_desc = "ONNX Hosting: host an ONNX model for inferencing with ONNXRuntime";
   std::string model_path;
   std::string address = "0.0.0.0";
-  int http_port = 5000;
+  int http_port = 8001;
   int num_http_threads = std::thread::hardware_concurrency();
 
  private:
-  // Print help and exit if there is a bad value
+  // Print help and return if there is a bad value
   Result ValidateOptions() {
     if (num_http_threads <= 0) {
       PrintHelp(std::cerr, "num_http_threads must be greater than 0");
       return Result::ExitFailure;
     } else if (http_port < 0 || http_port > 65535) {
       PrintHelp(std::cerr, "http_port input invalid");
       return Result::ExitFailure;
+    } else if (!file_exists(model_path)) {
+      // TODO: check filetypes ?
```

Comment:
I don't think we could check file type unless load it into ONNX Runtime. I think check if the file exists should be good enough for here.

Reply:
Agreed. Removed